packages/webcomponent/test/JsonFormsElement.test.ts

```diff
@@ -251,7 +251,7 @@ test.cb('Connect JSON Forms element and cause re-init store', t => {
     const verticalLayout1 = jsonForms.children.item(0);
     t.is(verticalLayout1.className, 'layout');
     t.is(verticalLayout1.children.length, 1);
-    const initState: JsonFormsState = {
+    const initState2: JsonFormsState = {
```

Comment:
Is this necessary?

Reply:
Yes, because of name shadowing (it was a warning previously to the change).